airgun/session.py

```diff
@@ -217,7 +217,7 @@ def __exit__(self, exc_type, exc_value, traceback):
         passed = True if exc_type is None else False
         try:
             if passed:
-                self.login.logout()
+                pass
             else:
                 self.take_screenshot()
```

Comment:
```python
try:
    if not passed:
        self.take_screenshot()
except Exception as err:
```

Also, can you add issue with link to this change that says "we used to call `logout()` at the end of each test (which did nothing, as logout was not implemented), but since commit foobar we no longer do. See also discussion in #397 . Let's continue discussion here - should we call logout on end of each test?".

Reply:
I think we should not call the logout for each test, but we should have log out functionality as in because there are test scenarios need to verify that.

Reply:
@mirekdlugosz I agree with omkar, It is not required to call logout in each test case if we add it, the execution time has drastically changed and that will not be a good sign for us.
As we know the current logout method did nothing it just calls and passes the logout without performing the actual logout. 
I think we can call the logout method when it is actually required and this way, we can achieve what we want without impacting the performance.
What do you say?

Reply:
Guys, I have not asked on opinion on logout on end of each test. I have asked to create issue where such discussion could take place. So this way we capture the initial intent of Airgun authors and can come up with more data when timing is better. I do not want to block this PR on that discussion and decision.

I have now created that issue: #475

Reply:
As you mentioned **Let's continue the discussion here - should we call logout on the end of each test?".** I personally thought you wanted to discuss on this same thread.